ThinDownloadManager/src/main/java/com/thin/downloadmanager/DownloadRequest.java

```diff
@@ -211,6 +221,14 @@ public boolean getDeleteDestinationFileOnFailure() {
         return mDeleteDestinationFileOnFailure;
     }
 
+    public DownloadRequest setStreamingDownload(boolean isStreamingEnabled) {
+        this.isStreamingEnabled = isStreamingEnabled;
+        return this;
+    }
+
+    public boolean isStreamingDownloadEnabled() {
```

Comment:
Could we rename this api to `isResumable()` ? Also could add a method comment

Reply:
Done: Changed to `isResumable()`